examples/mat-mat-perf.cpp

```diff
@@ -160,14 +158,14 @@ __global__ void matMultKernel2(int N, double* C, double* A, double* B)
     __syncthreads();
 
     for (int n = 0; n < CUDA_BLOCK_SIZE; ++n)
-      Cs[threadIdx.y][threadIdx.x] += As[threadIdx.y][n] * Bs[n][threadIdx.x];
+      dot += As[threadIdx.y][n] * Bs[n][threadIdx.x];
```

Comment:
I think this should be a different variant, we need to figure how to make this portable using RAJA.